package.json

```diff
@@ -11,7 +11,7 @@
     "dashboarding"
   ],
   "private": true,
-  "version": "1.3.5",
+  "version": "1.3.2",
```

Comment:
Why do we need to rollback the version? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/1.3/package.json